SConstruct

```diff
@@ -277,6 +277,15 @@ options.Add(
 
 # general variables
 
+options.Add(
+	BoolVariable(
+		"GAFFERCORTEX",
+		"Builds and installs the GafferCortex modules. These are deprecated and will "
+		"be removed completely in a future version.",
+		True
```

Comment:
I think it'd be fair to default this off and then go flip it on in `config/ie/options` as well as the travis tests. But I'll merge as is and you can make a followup PR with that if you think its worth while.